diff --git gimp-22100/gimp.pc.in gimp-2.10.0/gimp.pc.in
index 9cf1da9..eca9650 100644
--- gimp-22100/gimp.pc.in
+++ gimp-2.10.0/gimp.pc.in
@@ -12,6 +12,6 @@ gimplocaledir=@gimplocaledir@
 Name: GIMP
 Description: GIMP Library
 Version: @GIMP_REAL_VERSION@
-Requires: gdk-pixbuf-2.0 >= @GDK_PIXBUF_REQUIRED_VERSION@ cairo >= @CAIRO_REQUIRED_VERSION@ gegl-0.3 >= @GEGL_REQUIRED_VERSION@
+Requires: gdk-pixbuf-2.0 >= @GDK_PIXBUF_REQUIRED_VERSION@ cairo >= @CAIRO_REQUIRED_VERSION@ gegl >= @GEGL_REQUIRED_VERSION@
 Libs: -L${libdir} -lgimp-@GIMP_API_VERSION@ -lgimpmath-@GIMP_API_VERSION@  -lgimpconfig-@GIMP_API_VERSION@ -lgimpcolor-@GIMP_API_VERSION@ -lgimpbase-@GIMP_API_VERSION@ @RT_LIBS@
 Cflags: -I${includedir}/gimp-@GIMP_API_VERSION@
